challenge.md

```diff
@@ -6,37 +6,37 @@ Write your answers inside this file, where it's indicated by the comments.
 
 1. We just forked a repo on GitHub.com and want to start working on it locally. What command do we use to do that?
 
-<!-- Answer Starts Here -->
+git clone
 
 <!-- Answer Ends Here -->
 
 2. OK, we just wrote some code. What command could we use to see a list of all the changes that have been made since the last commit?
 
-<!-- Answer Starts Here -->
+git diff
 
 <!-- Answer Ends Here -->
 
 3. Oops - it looks like there was a change that we forgot to include in our last commit. How can we revise that commit and fix things?
 
-<!-- Answer Starts Here -->
+git commit --amend
 
 <!-- Answer Ends Here -->
 
 4. It looks like there was a change on the original repo that we forked from. How could we grab those changes and incorporate them into our local repo?
 
-<!-- Answer Starts Here -->
+Get changes from upstream. Use git pull upstream?
```

Comment:
Yep! We'd need to add an upstream (`git remote add upstream [some_url]`), but once we do, we can pull from it using `git pull upstream` (plus, optionally, a branch).